lib/index.js

```diff
@@ -192,27 +153,13 @@ export default {
           return [];
         }
 
-        // Construct the list of linters to be passed to the CLI by looking at
-        // the linters made available for our language grammar by the LinterMap,
-        // and whether or not the engines are enabled in a user's config file.
-        if (Object.prototype.hasOwnProperty.call(linterMap, grammarName) === true) {
-          // for (const linter of linterMap[grammarName]) {
-          linterMap[grammarName].forEach((linter) => {
-            linterNames.add(linter);
-          });
-        }
-        const configEnabledEngines = getEnabledEngines(configurationFilePath);
-        const linterEnabledEngines = Array.from(linterNames).filter(
-          linterName => configEnabledEngines.has(linterName));
-
         // Construct the command line invocation which runs the Code Climate CLI
         const relpath = atom.project.relativizePath(filePath).pop();
-        const execArgs = ['analyze', '-f', 'json'].concat(
-          getEngineArgs(linterEnabledEngines),
-          [relpath, '</dev/null']);
+        const execArgs = ['analyze', '-f', 'json', relpath];
         const execOpts = {
           cwd: dirname(configurationFilePath),
           uniqueKey: `linter-codeclimate::${relpath}`,
+          env: process.env,
```

Comment:
@pointlessone Yes, that's why I've included the output **without** `</dev/null` too, and I can assure that **the tests fail because of the new codeclimate cli wrapper**. It's not an issue of `linter-codeclimate` but `codeclimate cli`.

I you take a look at the CI log, codeclimate is installed at the very beginning of the tests with:

```shell
$ git clone --depth 1 https://github.com/codeclimate/codeclimate.git
$ cd codeclimate && sudo make install && cd ..
```

So it clones the `master` branch of the repo, which is the latest one (v0.63.1). Tests fail with v0.63.0 and v0.63.1. You can clone the repo with the tag `v0.62.0` and you'll see that the tests succeed (and the plugin itself in dev mode).

The `</dev/null` was a comment because since codeclimate cli v0.63.0, it has stopped working, so anyone using this plugin with codeclimate cli v0.63.0+ will think that the problem is related to the plugin, but apparently it is not.